homeworks/artem.sokur_Grimdvl/PokemonProject/pokemons.html

```diff
@@ -0,0 +1,365 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://fonts.google.com/share?selection.family=Roboto:wght@100;300;700">
+        <link rel="stylesheet" href="style/style.css">
+        <link rel="stylesheet" href="style/pokemons.css">
+        <title>PokemonProject</title>
+    </head>
+    <body>
+        <header>
+            <div class="container" style="background-color: #333;">
+                <section class="head">
+                    <a class="head__logo" href="https://www.mastersacademy.education/" target="_blank">
+                        <img class="head__logo-img" src="img/icons/logo-ma.svg" alt="masters-academy">
+                        <span class="head__logo-title">MASTERS ACADEMY</span>
+                    </a>
+    
+                    <nav class="head__navigation">
+                        <a class="head__navigation-item active" href="#">Pokemons</a>
+                        <a class="head__navigation-item" href="index.html">About</a>
+                        <a class="head__navigation-item" href="login.html">Login</a>
+                    </nav>
+                </section>
+            </div>
+
+            <div class="container" style="background-color: #221F1F;">
+                <section class="filters">
+                    <form class="filters__drop">
+                        <input type="checkbox" id="drop-toggle" class="filters__drop-toggle">
+                        <label for="drop-toggle" class="filters__drop--btn">Type</label>
+                        <ul class="filters__drop__list">
+                            <li class="filters__drop__list-type">
+                                <a href="#">Fighting</a>
+                            </li>
+                            <li class="filters__drop__list-type">
+                                <a href="#">Dark</a>
+                            </li>
+                        </ul>
```

Comment:
It's not correct! you need use <select> for this. Pls read about <select> and fix this.